api/namex/resources/exact_match.py

```diff
@@ -24,10 +24,10 @@ def get():
               'sow=false' + \
               '&df=name_exact_match' + \
               '&wt=json' + \
-              '&q=' + query
-        results = requests.get(url)
-        text = results.text
-        answer = json.loads(text)
+              '&q=' + urllib.parse.quote(query)
+        print(url)
```

Comment:
consider using *current_app.logger.debug* (or info, warning, error) so that it logs out correctly on the pods.